app/javascript/lib/metrics-perf.js

```diff
@@ -171,19 +181,63 @@ export function calculatePerfTimes(perfTimes) {
       // so, e.g. the cluster plot will wait until after the expression data is fetched, and then render itself
       // based on a subset of that data
       'perfTime:frontend:other': now - perfTimes.requestStart - perfTimes.plot,
-      'perfTime:frontend:plot': perfTimes.plot,
-      'cache': 'client'
+      'perfTime:frontend:plot': perfTimes.plot
     }
     const perfProps = roundValues(Object.assign({}, rawPerfProps))
     perfProps['perfTime:url'] = perfTimes.url
+
```

Comment:
This calculatePerfTimes function is now too long-- I'm not sure the best places to slice out helper functions, but we definitely need some

Reply:
Indeed -- I refactored in 8143beb and b00b600.